test/src/main/java/org/apache/accumulo/test/ThriftServerBindsBeforeZooKeeperLockIT.java

```diff
@@ -94,7 +94,7 @@ public void testMonitorService() throws Exception {
             final int responseCode = cnxn.getResponseCode();
             final String errorText = FunctionalTestUtils.readAll(cnxn.getErrorStream());
             // This is our "assertion", but we want to re-check it if it's not what we expect
-            if (HttpURLConnection.HTTP_UNAVAILABLE == responseCode && null != errorText && errorText.contains(BasicServlet.STANDBY_MONITOR_MESSAGE)) {
+            if (HttpURLConnection.HTTP_UNAVAILABLE == responseCode && null != errorText && errorText.contains("This is not the active Monitor")) {
```

Comment:
Is this text stored in a template now? Please leave a comment for future devs in this test case.